src/redux/actions.js

```diff
@@ -55,29 +52,30 @@ export var student = {
     outreach: 0,
     professor: 0,
     staff: 0,
-    misc: 0
+    misc: 0,
   },
-  role: AuthStates.STUDENT
+  role: AuthStates.STUDENT,
+  uid: '',
 };
 
-export var guest = {
-  role: AuthStates.GUEST
+export const guest = {
+  role: AuthStates.GUEST,
 };
 
 // action creators
 
-export const updateUser = (user) => {
+export function updateUser(user) {
```

Comment:
This is more consistent with what I've used elsewhere. I typically use the `function` syntax for named functions and the arrow function syntax for anonymous functions. I can change it back if it doesn't mesh with the rest of the style we're using.